man.Rmd

```diff
@@ -217,10 +284,24 @@ It's a little extra work keeping it all up to date, but the extra effort is ofte
 #' `grepl(pattern, string)`.
 ```
 
-If you want to use multiple paragraphs or a bulleted list, you can use the explicit `@description` tag[^man-4].
-Here's an example from `stringr::str_like()`, which mimics the `LIKE` operator from SQL:
+If you want more than one paragraph, you must use an explicit `@description` tag to prevent the second (and subsequent) paragraphs from being turned into the `@details`.
+Here's a two-paragraph `@description` from `stringr::str_view()`:
 
-[^man-4]: You can also use explicit `@title` and `@details` tags if needed, but we don't generally recommend them because they add extra noise to the docs without enabling any extra functionality.
+```{r}
+#' View strings and matches
+#'
+#' @description
+#' `str_view()` is used to print the underlying representation of a string and
+#' to see how a `pattern` matches.
+#'
+#' Matches are surrounded by `<>` and unusual whitespace (i.e. all whitespace
+#' apart from `" "` and `"\n"`) are surrounded by `{}` and escaped. Where
+#' possible, matches and unusual whitespace are coloured blue and `NA`s red.
+```
+
+Here's another example from `stringr::str_like()`, which has a bullet list in `@description`:[^man-10]
+
+[^man-10]: The markdown parser used by roxygen2 does not actually require an empty line before lists, but it's allowed and some people prefer that style.
```

Comment:
This is how I handled the fact that the empty line is not necessary and our examples sometimes have it and sometimes don't.

Reply:
To me this feels like too much inside baseball for even a footnote.

Reply:
I'm deleting the footnote. This factoid is currently in the section I just wrote on the most important markdown stuff, so we'll see what you think.